src/controllers/index.js

```diff
@@ -60,3 +62,13 @@ exports.status = async (req, res) => {
 		console.log(err)
 	}
 }
+
+exports.sessionUpdate = async (req, res) => {
+	try {
+		console.debug(JSON.stringify(req.body, null, '\t'))
+		res.status(200).send(responses.success_ack)
```

Comment:
This idea of the controller immediately responding with a success acknowledgement doesn't apply to non-dsep or internal calls. Since this is an internal call from mentoring service to bpp, we are no longer dealing with DSEP protocol here. Hence no need to give early response. Here use the regular response flow where success or failure responses are given based on the execution of controller or service logic.